### PR TITLE
Publishing doc on GitHub pages on every commit on master

### PR DESCRIPTION
**Pull request by Lucas-C**
GitHub Actions test pipeline: https://github.com/Lucas-C/pyfpdf/actions/runs/461706850

Result preview: https://lucas-c.github.io/pyfpdf/

### CODE REVIEW

**Comment by Lucas-C** [comment]
This is not quite ready yet, I need to solve this error:

      ModuleNotFoundError: No module named 'codehilite(linenums=False)'

**Comment by Lucas-C** [comment]
Problem solved!
Ready for review :)
Here is the resulting website preview: https://lucas-c.github.io/pyfpdf/

File: PyPIReadme.rst [hunk 1]
```diff
@@ -47,17 +47,17 @@ installers. For example, using pip:
...
--  `Reference Manual <https://alexanderankin.github.io/pyfpdf/mkdocs_docs/ReferenceManual/index.html>`__
+-  `Read the Docs <https://alexanderankin.github.io/pyfpdf/>`__
+-  `FAQ <https://alexanderankin.github.io/pyfpdf/FAQ/index.html>`__
+-  `Tutorial <https://alexanderankin.github.io/pyfpdf/Tutorial/index.html>`__ (Spanish translation available)
+-  `Reference Manual <https://alexanderankin.github.io/pyfpdf/ReferenceManual/index.html>`__
 
 For further information, see:
 
 -  `documentation home <https://alexanderankin.github.io/pyfpdf/>`__
 -  `epydocs documentation home <https://alexanderankin.github.io/pyfpdf/epydocs>`__
```
Comment (alexanderankin) [inline PyPIReadme.rst:58]: this should also get removed

**Comment by alexanderankin** [comment]
This seems to be what is needed. I would merge if i could push a commit to the branch it was coming from.

File: PyPIReadme.rst (hunk 1 above)
Comment (Lucas-C) [reply PyPIReadme.rst]: Well spotted!

**Comment by Lucas-C** [comment]
I added a commit fixing the URL you spotted and adding a redirect page from old URLs to the new ones

**Comment by Lucas-C** [comment]
@alexanderankin seems like the doc wasn't properly updated after the merge...

https://alexanderankin.github.io/pyfpdf/

Could you please check in GitHub settings that the branch configured to host GitHub Pages is `gh-pages` ?

**Comment by alexanderankin** [comment]
@Lucas-C yep made the change.

**Comment by Lucas-C** [comment]
Awesome, looks good! -> https://alexanderankin.github.io/pyfpdf/

**Comment by Lucas-C** [comment]
The redirections do not work though: https://alexanderankin.github.io/pyfpdf/mkdocs_docs/

**Pull request merged**